File: TaskManager/js/src/models/entities/employee.js
// Employee класс для представления сущности сотрудника
export class Employee {
    constructor(id, lastname, firstname, middlename, position, phoneNumber, email) {
        this.ID = id
        this.lastname = lastname
        this.firstname = firstname
        this.middlename = middlename
        this.position = position
        this.phoneNumber = phoneNumber
        this.email = email
    }
}

File: TaskManager/app/models/entities/Employee.go
package entities

// Employee структура сущности сотрудника
type Employee struct {
	ID          int64  `json:"id"`           // идентификатор
	Lastname    string `json:"lastname"`     // фамилия
	Firstname   string `json:"firstname"`    // имя
	Middlename  string `json:"middlename"`   // отчество
	Position    string `json:"position"`     // должность
	PhoneNumber string `json:"phone_number"` // телефонный номер
	Email       string `json:"email"`        // почтовый адрес
}
